builtin/providers/aws/resource_aws_security_group.go

```diff
@@ -723,6 +766,10 @@ func matchRules(rType string, local []interface{}, remote []map[string]interface
 				if ok {
 					numRemoteCidrs = len(r["cidr_blocks"].([]string))
 				}
+				riRaw, ok := r["ipv6_cidr_blocks"]
+				if ok {
+					numRemoteIpv6Cidrs = len(r["ipv6_cidr_blocks"].([]string))
```

Comment:
Followed the same code as `cidr_blocks` above and that doesn't panic so i think we are safe